### DISCUSSION TITLE
How to turn automatic syncing back on

### DISCUSSION THREAD

**Discussion by SeibertronSS** [argoproj/argo-cd#10001] [Q&A]
I found out that after using the rollback feature, the app's auto-sync will be turned off, how do I turn it back on.

**Comment by mubarak-j** [comment]
do you have [auto sync](https://argo-cd.readthedocs.io/en/stable/user-guide/auto_sync/#automated-sync-policy) enabled? 
```yaml
spec:
  syncPolicy:
    automated: {}
```
it's also possible to enable in ArgoCD UI under App details:
![argocd-ui](https://user-images.githubusercontent.com/83465122/179269976-8c887c98-4d9c-4cd0-bd80-feee8ffec764.png)
